src/components/fileviewer.js

```diff
@@ -224,17 +224,19 @@ const Line = ({ lineNumber, lineText, coverage, selectedLine, onLineClick }) =>
 };
 
 /* This component contains metadata of the file */
-const FileViewerMeta = ({ revision, path, status, coverage }) => {
+const FileViewerMeta = ({ revision, path, self, coverage }) => {
   return (
     <div className="file-meta-viewer">
       <div className="file-meta-center">
         <div className="file-meta-title">File Coverage</div>
         <CoveragePercentageViewer
           coverage={coverage}
         />
-        <status/>
+        <FileViewerStatus ref={
+          c => { self.status = c }
+        } />
```

Comment:
There is a magic property called `ref` which is called after mounting, which is called with the newly created component.  This is the point which you can assign it to `this.status` so other code can use it.